lib/process.js

```diff
@@ -57,6 +63,20 @@ ProcessInfo.buildProcessTree = function (infos) {
   return treeRoot
 }
 
+ProcessInfo.prototype.getCoverageMap = function (merger) {
```

Comment:
This is cool! it works because we already output the coverage for each subprocess to a separate file named after the pid?